src/poetry/core/json/__init__.py

```diff
@@ -24,7 +26,7 @@ def validate_object(obj: dict, schema_name: str) -> List[str]:
     from jsonschema import Draft7Validator
 
     validator = Draft7Validator(schema)
-    validation_errors = sorted(validator.iter_errors(obj), key=lambda e: e.path)
+    validation_errors = sorted(validator.iter_errors(obj), key=lambda e: e.path)  # type: ignore
```

Comment:
Could you add the error code to `# type: ignore`? 

I think we should add the `python-check-blanket-type-ignore` check to `pre-commit` like we have in `poetry`.

Reply:
I'll add the error code